clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -698,23 +698,45 @@ public void subscribe(List<String> topics, ConsumerRebalanceListener listener) {
     }
 
     /**
+     * Subscribe to the givent list of topics. See {@link #subscribe(Collection, ConsumerRebalanceListener)}
+     * for more details.
+     * @param topics The list of topics to subscribe to
+     * @param listener Non-null listener instance to get notifications on partition assignment/revocation for the
+     *                 subscribed topics
+     */
+    @Override
+    public void subscribe(List<String> topics, ConsumerRebalanceListener listener) {
+        subscribe(topics, listener);
+    }
+
+
+    /**
      * Subscribe to the given list of topics to get dynamically assigned partitions.
      * <b>Topic subscriptions are not incremental. This list will replace the current
      * assignment (if there is one).</b> It is not possible to combine topic subscription with group management
-     * with manual partition assignment through {@link #assign(List)}.
+     * with manual partition assignment through {@link #assign(Collection)}.
      *
      * If the given list of topics is empty, it is treated the same as {@link #unsubscribe()}.
      *
      * <p>
-     * This is a short-hand for {@link #subscribe(List, ConsumerRebalanceListener)}, which
+     * This is a short-hand for {@link #subscribe(Collection, ConsumerRebalanceListener)}, which
      * uses a noop listener. If you need the ability to either seek to particular offsets, you should prefer
-     * {@link #subscribe(List, ConsumerRebalanceListener)}, since group rebalances will cause partition offsets
+     * {@link #subscribe(Collection, ConsumerRebalanceListener)}, since group rebalances will cause partition offsets
      * to be reset. You should also prefer to provide your own listener if you are doing your own offset
      * management since the listener gives you an opportunity to commit offsets before a rebalance finishes.
      *
      * @param topics The list of topics to subscribe to
      */
     @Override
+    public void subscribe(Collection<String> topics) {
+        subscribe(topics, new NoOpConsumerRebalanceListener());
+    }
+
+    /**
+     * Subscribe to the givent list of topics. See {@link #subscribe(Collection)} for more details.
+     * @param topics The list of topics to subscribe to
+     */
+    @Override
     public void subscribe(List<String> topics) {
         subscribe(topics, new NoOpConsumerRebalanceListener());
```

Comment:
Yes indeed, I pushed before the end of the unit tests by error.
Now fixed, I'm happy to rebase once the PR is in good shape if you want a single commit.